public/manifest.json

```diff
@@ -1,19 +1,25 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "ReactPlay",
+  "name": "ReactPlay",
   "icons": [
     {
       "src": "favicon.ico",
       "sizes": "64x64 32x32 24x24 16x16",
       "type": "image/x-icon"
     },
+    {
+      "src": "maskable_icon.png",
+      "sizes": "64x64 32x32 24x24 16x16 512x512 192x192",
+      "type": "image/png",
+      "purpose": "maskable"
+    },
     {
       "src": "logo192.png",
```

Comment:
make sure this is also our official reactplay icon.

Reply:
Alright, any link to the brand assets for the different logo sizes?